dangerzone/logic.py

```diff
@@ -34,7 +34,8 @@ def __init__(self, isolation_provider: IsolationProvider) -> None:
 
         # Languages supported by tesseract
         with open(get_resource_path("ocr-languages.json"), "r") as f:
-            self.ocr_languages = json.load(f)
+            unsorted_ocr_languages = json.load(f)
+            self.ocr_languages = dict(sorted(unsorted_ocr_languages.items()))
```

Comment:
I come from a Python 2 background, so `dict(sorted(...))` seemed dangerous to me, as there are no guarantees that the `dict()` constructor would respect the provided order. Turns out that this is the case on Python 3.7+, so we're good: https://stackoverflow.com/a/613218. Just wanted to point that out.

Reply:
Yes, it also felt a bit odd to me since dictionaries have no order, supposedly. But it did sort by keys. Thanks for looking into it!